src/helm/benchmark/run_spec_factory.py

```diff
@@ -130,30 +121,10 @@ def alter_run_spec(run_spec: RunSpec) -> RunSpec:
             chatml_expander = ChatMLRunExpander()
             run_spec = singleton(chatml_expander.expand(run_spec))
 
-        # Anthropic prompts
+        # Anthropic Claude 1 and 2 prompts
         if ANTHROPIC_CLAUDE_1_MODEL_TAG in model.tags or ANTHROPIC_CLAUDE_2_MODEL_TAG in model.tags:
             run_spec = singleton(AnthropicClaude2RunExpander().expand(run_spec))
 
-        # Anthropic prompts
-        if ANTHROPIC_CLAUDE_3_MODEL_TAG in model.tags:
```

Comment:
If we already have results for the multimodal models, do we have to rerun everything?

Reply:
I'm not sure. You will have to check the prompts in your existing Claude 3 runs, and decide if they are what you want or if you want to rerun them.

Reply:
As discussed offline - the multimodal adapters apparently ignore `global_prefix` and `global_suffix`. So the multimodal evaluations are unaffected.